src/lib/thanos/networks.ts

```diff
@@ -1,28 +1,35 @@
 import { ThanosNetwork } from "lib/thanos/types";
+import { getMessage } from "lib/ui/i18n";
 
 export const NETWORKS: ThanosNetwork[] = [
   {
     id: "mainnet",
-    name: "Tezos Mainnet",
-    description: "Carthage mainnet",
+    name: getMessage("tezosMainnet"),
```

Comment:
Usually, when the function is executed, locales files aren't loaded yet, so the only reason of using `getMessage` here is preventing hardcoding